src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PickaxeOfTheSeeker.java

```diff
@@ -27,7 +27,7 @@ public ItemInteractionHandler getItemHandler() {
 				for (int x = -4; x <= 4; x++) {
 					for (int y = -4; y <= 4; y++) {
 						for (int z = -4; z <= 4; z++) {
-							if (p.getLocation().getBlock().getRelative(x, y, z).getType().toString().endsWith("_ORE") && closest == null || p.getLocation().distance(closest.getLocation()) < p.getLocation().distance(p.getLocation().getBlock().getRelative(x, y, z).getLocation())) {
+							if (p.getLocation().getBlock().getRelative(x, y, z).getType().toString().endsWith("_ORE") && (closest == null || p.getLocation().distance(closest.getLocation()) < p.getLocation().distance(p.getLocation().getBlock().getRelative(x, y, z).getLocation()))) {
```

Comment:
Another thing: If you wanna optimize the performance, use distanceSquared instead of distance